ui/src/message_popup/data_lists.jsx

```diff
@@ -136,7 +136,36 @@ export const allStudents = [
 ];
 
 export const allQuestions = [
-  { studentId: 4, text: 'At the conclusion of your lesson plan for this challenge, you seed a group discussion by asking "What are you curious about related to photosynthesis?"  Hayin says "Why are plants green?"  What do you do?' },
-  { studentId: 10, text: `Imagine a new student, Sasha, joins your classroom during the lesson plan you developed for this challenge.  As part of helping her feel comfortable, you pull her aside and give her a quick overview of what's happening at the beginning of the class.  She interrupts and asks, "I do better with visuals, can you draw me a picture of photosynthesis?"  What could you quickly sketch to directly answer to her question?` },
-  { studentId: 2, text: 'In the context of the lesson plan you developed for this challenge, Floyd says "why are we even doing this?"  Respond in way that engages his natural curiosity and tendency towards asking questions' }
+  { 
+    studentId: 4, 
+    text: 'At the conclusion of your lesson plan for this challenge, you seed a group discussion by asking "What are you curious about related to photosynthesis?"  Hayin says "Why are plants green?"  What do you do?' 
+  },
+  {
+    studentId: 10, 
+    text: `Imagine a new student, Sasha, joins your classroom during the lesson plan you developed for this challenge.  As part of helping her feel comfortable, you pull her aside and give her a quick overview of what's happening at the beginning of the class.  She interrupts and asks, "I do better with visuals, can you draw me a picture of photosynthesis?"  What could you quickly sketch to directly answer to her question?` 
+  },
+  { 
+    studentId: 2, 
+    text: 'In the context of the lesson plan you developed for this challenge, Floyd says "why are we even doing this?"  Respond in way that engages his natural curiosity and tendency towards asking questions' 
+  },
+  {
+    studentId: 8,
+    text: 'Imagine in the context of the lesson plan you developed for this challenge, there is an activity where students are coming up with questions to investigate.  Ada says "How many questions should I write and what do you want me to include in them?"  Respond in a way that draws out the student’s curiosity and pushes them towards asking questions that are meaningful to them.'
+  },
+  {
```

Comment:
This looks like it might be duplicate of https://github.com/mit-teaching-systems-lab/threeflows/pull/16/files#diff-2fff7c84f93958ba0c569fcfbc733498R143